resample/bootstrap.py

```diff
@@ -144,19 +142,25 @@ def confidence_interval(
 
     Notes
     -----
+    Bootstrap interval estimates are valid for a broad range of common applications,
+    such as when estimating means, variances, correlations, or regression coefficients.
+    However, keep in mind that there are situations where the bootstrap will give
```

Comment:
How do I make these nice commit-able suggestions that you often use @dsaxton ?

I would write the part starting with "However..." like this:

The non-parametric bootstrap may give misleading results for values of alpha that are very close to 1.0, since there are by nature very few observations in the tails. In this case, the parametric bootstrap would be preferred, which does not suffer from these issues.

Reply:
Otherwise looking good!

Reply:
When you comment you can click the + / - symbol to the right of "Preview" and it'll let you make inline suggestions.

Hmm, I'm more thinking about _parameters_ (fixed properties of the underlying distribution we're sampling from) that we won't be able to find interval estimates for using the bootstrap, irrespective of any confidence level. The distribution maximum comes to mind since the bootstrap replicates will always be bounded from above by that value, so naively taking percentiles (say) won't work.

When we say there are few observations in the tails, wouldn't that not necessarily be true since when talking about alpha we're dealing with the empirical distribution of replicates, so we can actually control the number of observations through the size argument?

Reply:
You can bootstrap the sample maximum just fine if you do a parametric bootstrap. See Efron's book.